File: src/main/java/me/rsman/BetterMinecraftCore/Tasks/PlayerTasks.java
package me.rsman.BetterMinecraftCore.Tasks;

import me.rsman.BetterMinecraftCore.BetterMinecraftCore;
import me.rsman.BetterMinecraftCore.Managers.PlayerManager;

public class PlayerTasks {
    public static void updatePlayers(){
        //long startTime = System.nanoTime();
        BetterMinecraftCore.getInstance().getServer().getOnlinePlayers().forEach(PlayerManager::updatePlayerAttributes);
        //long endTime = System.nanoTime();
        //FishingChill.getInstance().getLogger().info("inventory analyzing system took " + ((endTime - startTime)));
    }
}

File: src/main/kotlin/me/rsman/BetterMinecraftCore/Tasks/PlayerTasks.kt
package me.rsman.BetterMinecraftCore.Tasks

import me.rsman.BetterMinecraftCore.BetterMinecraftCore.Companion.instance
import org.bukkit.entity.Player
import me.rsman.BetterMinecraftCore.Managers.PlayerManager

object PlayerTasks {
    @JvmStatic
    fun updatePlayers() {
        instance.server.onlinePlayers.forEach { player: Player -> PlayerManager.updatePlayerAttributes(player) }
    }
}